app/routers/users_router.py

```diff
@@ -1,17 +1,101 @@
 from typing import List
 
-from fastapi import Depends, APIRouter
+from fastapi import Depends, APIRouter, HTTPException
+from pydantic import ValidationError
 from sqlalchemy.orm import Session
-
-from app import crud
-from app.dependencies import get_db
+from sqlalchemy.exc import IntegrityError
+from app import crud, schema
+from app.dependencies import get_db, get_current_user, have_user_permission
 from app.models.account_type import AccountType
-from app.schema import UserResponse
+from app.schema import UserResponse, User, UserEdit
 
 router = APIRouter(tags=["user"])
 
 
 @router.get("/users", response_model=List[UserResponse])
 async def get_organizations(skip: int = 0, limit: int = 100, db: Session = Depends(get_db)):
     users = crud.get_users_by_type(db, AccountType.ORGANIZATION, skip, limit)
+
+    return users
+
+
+@router.get("/admin/users", response_model=List[UserResponse], responses={
+    401: {
```

Comment:
Put this response to global variable

Reply:
I don't know if its a good idea because its only 1 record and create new variable will complicate the code more

Reply:
Idk but 1 think it will be good to extract 401 error to some variable because it will be used in future in others endpoint too.